src/main/scala/codacy/plugins/test/multiple/MultipleTests.scala

```diff
@@ -0,0 +1,90 @@
+package codacy.plugins.test.multiple
+
+import codacy.plugins.test._
+
+import com.codacy.analysis.core.model.{CodacyCfg, Configuration, FileCfg, FileError, FullLocation, Issue, LineLocation}
+import com.codacy.analysis.core.tools.Tool
+import com.codacy.plugins.api.results.Result
+import com.codacy.plugins.results.traits.{DockerToolDocumentation, ToolRunner}
+import com.codacy.plugins.runners.{BinaryDockerRunner, DockerRunner}
+import com.codacy.plugins.utils.BinaryDockerHelper
+import com.codacy.plugins.results.PluginResult
+
+import better.files._
+import java.io.{ File => JFile }
+
+import scala.util.{Failure, Success, Try}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/app/Codacy/codacy-plugins-test/pullRequest?prid=4500290)